src/main/java/com/devonfw/sample/archunit/general/dataaccess/violations/ThirdPartyDataMappingPersistanceOk.java

```diff
@@ -0,0 +1,8 @@
+package com.devonfw.sample.archunit.general.dataaccess.violations;
+
+import javax.persistence.Converter; // compliant
+
+@Converter
+public class ThirdPartyDataMappingPersistanceOk {
+
+}
```

Comment:
This is fine for a simple test of the check.
However, a reasonable example can be found here:
https://github.com/devonfw/devon4j/blob/master/documentation/guide-jpa.asciidoc#entities-and-datatypes

Reply:
I added a comment. So i resolve this now.